runtime/oti/vm_api.h

```diff
@@ -3115,10 +3116,11 @@ getStringUTF8Length(J9VMThread *vmThread,j9object_t string);
 *
 * @param[in] *vmThread current thread
 * @param[in] string the class name string
+* @param[in] expectedReturnCode the expected return code to the caller
```

Comment:
Same as above.

Reply:
Updated the documentation against the suggestion above.